lib/trln_argon/view_helpers/trln_argon_helper.rb

```diff
@@ -67,6 +67,16 @@ def consortium_long_name
         t('trln_argon.consortium.long_name')
       end
 
+      def holdings_list
```

Comment:
This is much closer. I would recommend considering a few more changes.

1. The method name `holdings_list` implies this is used in the display of holdings, but that is not the case. I might rename it something like `no_results_inst_display` so its purposes is clearer.
2. It looks like you're using `tap` where I think `reject` and `map` could be used instead.
3. I would also prefer it if the translation from code to display name happened before turning the array into a string. This can also then be accomplished in one method call instead of four.

Putting this altogether I might do something like this:

```      def no_results_inst_display
        TrlnArgon::Engine.configuration.sort_order_in_holding_list.split(', ')
                         .reject { |inst| inst =='trln' }
                         .map { |inst_code| t("trln_argon.institution.#{inst_code}.short_name") }
                         .to_sentence(words_connector: ', ', last_word_connector: ', or ')
      end
```

Reply:
Thanks for the comments! This is a very neat solution and I like it.